censys/search/v1/certificates.py

```diff
@@ -22,6 +24,16 @@ class CensysCertificates(CensysSearchAPIv1):
     """Max number of bulk requests."""
     bulk_path = f"/bulk/{INDEX_NAME}"
 
+    def __init__(self, **kwargs):
+        """Inits CensysCertificates.
+
+        See CensysSearchAPIv1 for additional arguments.
+
+        Args:
+            **kwargs: Arbitrary keyword arguments.
+        """
+        CensysSearchAPIv1.__init__(self, timeout=self.DEFAULT_TIMEOUT, **kwargs)
```

Comment:
We should make sure we don't override the user's timeout defined in kwargs.

Reply:
Nice catch